cuppa/src/test/java/org/forgerock/cuppa/reporters/DefaultReporterTest.java

```diff
@@ -103,7 +94,6 @@ public void reporterShouldLookGreatForFailingTests() {
                 "  1 failing",
                 "",
                 "  1) describe when failing test:",
-                "     java.lang.AssertionError:",
```

Comment:
This was failing on my machine. I assume it was because of different behaviours in different versions of dependencies.